legacy-tests/.eslintrc.json

```diff
@@ -10,6 +10,7 @@
   "rules": {
     "mocha/no-exclusive-tests": "error",
     "no-unused-expressions" : "off",
-    "no-new" : "off"
+    "no-new" : "off",
+    "prettier/prettier": "off"
```

Comment:
What does this do? (sorry have never used prettier)

Reply:
Prettier auto-formats code - trailing commas, tabs, semicolons, etc. It's pretty opinionated though so on codebases that haven't been using a formatter turning it on can introduce a lot of whitespace-type changes. This here to avoid churn by not running on the legacy tests.

Reply:
Ah, ok, so it's just disabling prettier entirely on the legacy tests :+1: